README.md

```diff
@@ -97,6 +97,23 @@ for both web and mobile.
 Currently we do not yet support persisted queries. This can be used in this starter kit currently only for web, but it is
 planed in the future.
 
+### Module Scaffolding with CLI
+
+Basic Module Scaffolding is provided with the following command:
```

Comment:
Maybe add some description about what is module, like:
"This starter kit encourages modular design of application features. Each feature should be designed as a decoupled module, deleting feature should ideally not break the remaining application. Basic feature module scaffolding is provided with the following command:"